### DISCUSSION TITLE
WILL COMPENSATE $$ -- My internet went down and afterwards, my slingbox IP cant be found

### DISCUSSION THREAD

**Discussion by lead2crm** [GerryDazoo/Slinger#598] [Q&A]
I also unplugged my router and plugged it back in. That didnt get my internet back up, but it came back online an hour later on its own, but my slingbox stopped working. Here's my error:

"8080 No valid slingbox ip info found in config.ini" --I never had to put one before. 

```
Using config file config.ini
Connection Manager Running on port 8080 with 10 max streams using URL slingbox.
Building page for Slingbox
Using built in default remote page definition.
Starting Streamer Thread for Slingbox
Streamer Running:  10 config.ini SLINGBOX 8080 8080 8388608
starting up on  port 8080 8388608
8080 Checking for slingbox at ('', 5201) 0
 * Serving Flask app 'slingbox_server'
 * Debug mode: off
8080 Error connecting to  ('', 5201)
Broadcast Responder Running
Server IP address is 192.168.86.247
8080 No valid slingbox ip info found in config.ini
8080 Finding Slingboxes on local network. My IP Info =  192.168.86.247
8080 Can't find a slingbox on network. Please make sure it's plugged in and connected. Check config.ini
8080 Giving up. Sorry...
```

**Comment by GerryDazoo** [comment]
The slingbox gives up trying to get an IP address via DHCP after about 30 minutes and then configures itself with a static IP that it hopes might work (192.168.1.254). Cycling the power on the slingbox should revive it.
WARNING: If you blocked tcp port 5678 Outgoing based on the slingbox IP address, the address it receives from your router after the power cycle might not be the same. So best to disconnect the uplink until you know for sure everything is OK.

**Reply by lead2crm** [reply]
Unfortunately, I powered down the Slingbox about an hour after my internet came back and powered it back up and got the same result from slingbox_server.exe

I didnt do anything myself to block 5678.

**Reply by lead2crm** [reply]
@GerryDazoo How can I send you a screenshot of my blocked ports? Would that be helpful? I didnt really want to post that here publicly.

**Reply by GerryDazoo** [reply]
Check the DHCP table in your router for the IP address of the slingbox. If it's not there you have more serious problems.  If you haven't blocked TCP port 5678 outgoing and you don't have a Pro-HD or 350/500 the chances you've bricked your box is very high. Although it's my experience that even the bricked boxes will still try to get a DHCP IP address after a power cycle.

**Reply by lead2crm** [reply]
I have a 500.

**Reply by sp222222** [reply]
are you using the 500 with wifi or wired ? I would try wired to make sure the 500 gets an IP address. you have been warned previously about blocking outbound 5678. I didn't think 500 could get bricked. Whatever you do, do NOT hard button reset the 500.

**Reply by lead2crm** [reply]
It is wired.

**Reply by lead2crm** [reply]
@GerryDazoo I did an ipconfig /all and I see a DHCP Server IP. What do I do now?

**Reply by lead2crm** [reply]
This part after "giving up. Sorry" is new...

![image](https://github.com/GerryDazoo/Slinger/assets/20845641/49804a38-3b44-4fc9-b7f2-7aca119866dd)

**Reply by GerryDazoo** [reply]
You have two copies of the server running at the same time.

**Reply by lead2crm** [reply]
Makes sense, I was trying this on my other computer remotely.

**Reply by lead2crm** [reply]
Okay, I'm back in the states and I have physical access to my server computer, my Slingbox and my router. Where should I start for debugging, now?

I have a hard time believing my Slingbox is fried because the outage I had was only for the internet provider, it wasn't a power outage--which would have made sense.

**Reply by lead2crm** [reply]
Any ideas?

**Comment by lead2crm** [comment]
@GerryDazoo 

Someone on the reddit form wrote:

> "You can reset te box by pressing the reset button on the back of the Slingbox. Hold the reset button as long as you notice al the leds in the middle of the front are flashing 2x long and 1x short, then release the button. It could take a while.  
> Then use the old windows slingplayer to configure your Slingbox again. This must be done within an LAN network. Disconnect your WAN cable (internet) The Slingplayer finds your Slingbox automaticly.  
> Just in case, block the complete ip range from Slingmedia if your want to use the Slingserver from Gerry over the internet.  
> Use for example iptables to do that.  
> iptables -I 'your firewall name' -m iprange --src-range 67.148.0.0-67.148.255.255 -j DROP  
> I run a webserver where you can download the old windows player:  
> https://84-80-239-248.fixed.kpn.net/download/SlingPlayer-Setup-EU-1.5.1.342.exe  
> Cheers, Marc (NL)"

Following this advice, I pressed the reset button on the back of my Slingbox 500 and then downloaded the latest update for the firmware (1.8.something) using my Ethernet cable. It shows me through the Slingbox setup screen (over the television) that Slingbox was assigned an IP address. I can't go much further because it want to search for a cable box model and such and it says "Network Error"

When i got to the old Slingbox app on my pc, it says, "Failed to load Slingplayer Desktop. Error code -118. Please check your internet connection and try again."

Now, I'm definitely stuck.

I don't understand the use "LAN not WAN", unless that means use your Ethernet cable and not your WiFi --my slingbox was always configured for Ethernet directly to the modem. This didn't work.

I then tried to connect my laptop directly (with wifi turned off on that laptop) to the Ethernet port on the Slingbox and use the windows program from there, but i got the similar error message, this time with an error code: -105. With wifi turned on and Ethernet plugged, i get error -118

Totally lost. I also noticed someone in this thread on GitHub said to not reset the Slingbox 500...that's definitely done. Am i screwed?

--Nick







> Check the DHCP table in your router for the IP address of the slingbox. If it's not there you have more serious problems. If you haven't blocked TCP port 5678 outgoing and you don't have a Pro-HD or 350/500 the chances you've bricked your box is very high. Although it's my experience that even the bricked boxes will still try to get a DHCP IP address after a power cycle.

**Comment by lead2crm** [comment]
**@GerryDazoo and anyone else that can help, I am willing to compensate for your assistance here or over remote session.**

Dear Community--

I am here in the states and leave again on Friday morning. I would like to be able to get this Slingbox up and running if possible before I leave. Can anyone provide guidance based on what I have provided in this story above?

Note: I have a Slingbox 500

--Nick
